lib/debug/server_dap.rb

```diff
@@ -685,6 +685,8 @@ def register_vars vars, tid
   end
 
   class ThreadClient
+    include SkipPathHelper
```

Comment:
The `ThreadClient` here is a patch to the one defined in `thread_client.rb`, so this should not be needed.

Reply:
Oh, you are right! Thanks for catching :)